README.md

```diff
@@ -654,6 +659,31 @@ Switched to a new branch 'daves'
 
 This will give you a local copy of the branch `daves`, and any update that has been pushed will also show up remotely.
 
+<a name="i-want-to-create-a-new-remote-branch-from-current-local-one"></a>
+### I want to create a new remote branch from current local one
+
+
+```sh
+$ git config push.default upstream
+$ git push -u origin HEAD
+```
+
+If you want to check out the other default configs which ```git push``` can take, visit the documentation for Git at https://git-scm.com/docs/git-config#git-config-pushdefault
+
+If you do not want to change the git configuration, you can also use:
+
+```sh
+$ git push -u <remote> HEAD
+```
+
+The following command will push all branches that exist locally and remotely with the same name:
```

Comment:
Thanks @Rijul1999 for your commit!
Just a last comment: this line should also be updated w.r.t. your last commit.
Indeed: whether the user runs `git config push.default upstream` or keeps the default config, it is inexact that "`git push` will push all branches that exist locally and remotely with the same name".

What happens is the following (feel free to adapt my phrasing):

> With the default mode and the `upstream` mode, `git push` will push the current branch w.r.t. the remote branch that has been registered previously with `-u`.
> (The behavior of the other modes of `git push` is described in the doc of `push.default`.)

Reply:
(FYI @Rijul1999 I just edited my latest comment)